data/transformed-tokens.json

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
This is generated by the token-transformer npm package for Style Dictionary build purposes